views/submit_request.erb

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="icon" type="image/x-icon" href="/assets/images/favicon.ico">
+    <link rel="stylesheet" href="/assets/css/madetech-frontend.css">
+    <link rel="stylesheet" href="/assets/css/styles.css">
+    <title>Requests - Made Tech</title>
+  </head>
+
+  <body>
+    <div class="top_bar">
+      <div class="top_bar__inner">
+        <div class="container">
+          <div class="row d-md-flex justify-content-center justify-content-md-between">
+            <div class="d-none d-md-block">Improving software delivery in every organisation</div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <header class="header">
+      <div class="header__inner">
+        <nav class="p-0 flex-column flex-lg-row navbar navbar-expand">
+          <div class="mr-lg-auto">
+            <div class="header__logo">
+              <a href="#" class="header__logo_link">
+                <div class="header_logo_type--by">
+                  <span class="header_logo_type__text">Requests</span>
+                  <span class="px-3">by</span>
+                  <span class="position-relative" style="top: -2px;">
```

Comment:
Would remove the inline style

Reply:
resolved in https://github.com/madetech/requestinator/commit/6211e1163bb0953f433e9a688b1a417fe181f785